test/test.api.js

```diff
@@ -223,6 +223,68 @@ DRIVERS.forEach(function(driverName) {
             });
         });
 
+        if (driverName === localforage.INDEXEDDB) {
```

Comment:
I don't see any `expect` calls in these tests so I don't see what they're testing for. If you want `called` to be more than one you'll want to test for its value with `expect`, which I think is what you're going for.